core/test/src/test_cpu.cpp

```diff
@@ -1129,7 +1133,23 @@ TEST_F(CpuImmediateTest, and_imm) {
 
     run_instruction(AND_IMM);
 }
+TEST_F(CpuZeropageTest, and) {
```

Comment:
https://en.cppreference.com/w/cpp/keyword/and 😁 I guess the macro attaches it to some other characters though.

Reply:
Yeah, should be safe :)